flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaUtils.java

```diff
@@ -206,79 +203,30 @@ private static void addBaseRemoteAkkaConfig(
+        final String outboundRestartBackoff =
+                configuration.getString(AkkaOptions.OUTBOUND_RESTART_BACKOFF);
 
         akkaConfigBuilder
                 .add("akka {")
                 .add("  actor {")
-                .add("    provider = \"akka.remote.RemoteActorRefProvider\"")
+                .add("    provider = remote")
                 .add("  }")
-                .add("  remote.artery.enabled = false")
-                .add("  remote.startup-timeout = " + startupTimeout)
                 .add("  remote.warn-about-direct-use = off")
                 .add("  remote.use-unsafe-remote-features-outside-cluster = on")
-                .add("  remote.classic {")
-                .add("    # disable the transport failure detector by setting very high values")
-                .add("    transport-failure-detector{")
-                .add("      acceptable-heartbeat-pause = 6000 s")
-                .add("      heartbeat-interval = 1000 s")
-                .add("      threshold = 300")
+                .add("  remote.artery {")
+                .add("    canonical.port = " + externalPort)
+                .add("    bind.port = " + port)
+                .add("    advanced {")
+                .add("      maximum-frame-size = " + akkaFramesize)
+                .add("      outbound-restart-backoff = " + outboundRestartBackoff)
+                .add("      tcp.connection-timeout = " + akkaTCPTimeout)
                 .add("    }")
-                .add("    enabled-transports = [\"akka.remote.classic.netty.tcp\"]")
-                .add("    netty {")
-                .add("      tcp {")
-                .add("        transport-class = \"akka.remote.transport.netty.NettyTransport\"")
-                .add("        port = " + externalPort)
-                .add("        bind-port = " + port)
-                .add("        connection-timeout = " + akkaTCPTimeout)
-                .add("        maximum-frame-size = " + akkaFramesize)
-                .add("        tcp-nodelay = on")
-                .add("        client-socket-worker-pool {")
-                .add("          pool-size-min = " + clientSocketWorkerPoolPoolSizeMin)
-                .add("          pool-size-max = " + clientSocketWorkerPoolPoolSizeMax)
-                .add("          pool-size-factor = " + clientSocketWorkerPoolPoolSizeFactor)
-                .add("        }")
-                .add("        server-socket-worker-pool {")
-                .add("          pool-size-min = " + serverSocketWorkerPoolPoolSizeMin)
-                .add("          pool-size-max = " + serverSocketWorkerPoolPoolSizeMax)
-                .add("          pool-size-factor = " + serverSocketWorkerPoolPoolSizeFactor)
-                .add("        }")
-                .add("      }")
-                .add("    }")
-                .add("    log-remote-lifecycle-events = " + logLifecycleEvents)
```

Comment:
Does not exist in Artery. What we can control in artery are these options:
```yaml
      # If this is "on", all inbound remote messages will be logged at DEBUG level,
      # if off then they are not logged
      log-received-messages = off

      # If this is "on", all outbound remote messages will be logged at DEBUG level,
      # if off then they are not logged
      log-sent-messages = off

      # Logging of message types with payload size in bytes larger than
      # this value. Maximum detected size per message type is logged once,
      # with an increase threshold of 10%.
      # By default this feature is turned off. Activate it by setting the property to
      # a value in bytes, such as 1000b. Note that for all messages larger than this
      # limit there will be extra performance and scalability cost.
      log-frame-size-exceeding = off
```

I did not added this yet, because there are multiple options here IMO:
1. Keep the old config option in Flink and apply that to both sent and received.
2. Deprecate the currently existing log option and add 2 for both received and sent.

We can also consider what to do with the frame size exceeding events.